airflow/providers/amazon/aws/hooks/glacier.py

```diff
@@ -35,9 +35,9 @@ class GlacierHook(AwsBaseHook):
         - :class:`airflow.providers.amazon.aws.hooks.base_aws.AwsBaseHook`
     """
 
-    def __init__(self, aws_conn_id: str = "aws_default") -> None:
-        super().__init__(client_type="glacier")
-        self.aws_conn_id = aws_conn_id
+    def __init__(self, *args, **kwargs) -> None:
+        kwargs.update({"client_type": "glacier", "resource_type": None})
+        super().__init__(*args, **kwargs)
```

Comment:
Accidentally found that `GlacierHook` only allows `aws_conn_id` and ignore other AwsBaseHook's parameters